web_src/js/webcomponents/GiteaOriginUrl.js

```diff
@@ -1,3 +1,5 @@
+import '@webcomponents/custom-elements';
```

Comment:
```suggestion
import '@webcomponents/custom-elements'; // imported for the side-effects
```

Reply:
Could you elaborate the "side-effects"? Otherwise future readers may not get the point.

Reply:
Well… You are not using this import for anything.
You only import it, and it does its magic automatically.
So, I simply want to dissuade future readers of this file from removing this line.
What about
```suggestion
import '@webcomponents/custom-elements'; // automatically adds custom elements for older browsers that don't support it
```
instead?

Reply:
OK (it's just how polyfill works)